client/src/pages/Home.tsx

```diff
@@ -359,10 +359,24 @@ function Home(this: any) {
     }, 1000);
   };
 
+  const clearInput = () => {
+    setFinalSummary("");
+    setInputValue("");
+    setSentimentText("");
+    setEmotionLabel("");
+    setSentimentScore(0);
+    setKeywords({});
+    setSentimentPlaceholder("");
+    setEmotionalPlaceholder("");
+    setWordCount(0);
+    setSelectedFont(null);
```

Comment:
Just wondering, do we want font to be reset?

All the other state makes sense, but IMO font choice should persist

Reply:
Grace's use case was for everything to revert back - so I took into account the font. Another thing i just thought of was the way the font option is presented is similar to every other feature so it holds no place above or below to be treated differently 🤷 .